src/networkConfig/index.ts

```diff
@@ -73,6 +74,32 @@ export const polygonMumbaiNetworkConfig = (
   providerUrl,
   subgraphUrl: 'https://api.studio.thegraph.com/query/49076/polygon-mumbai/v0.0.1',
   zeroExApiKey: config?.zeroExApiKey ?? '',
+  zeroExApiUrl: 'https://mumbai.api.0x.org',
+  apiUrlBase: 'https://api.encryptafile.com',
+});
+
+export const polygonMainnetNetworkConfig = (
+  providerUrl: string,
+  config?: { polygonMainnetApiKey?: string; zeroExApiKey?: string }
+): SarcoNetworkConfig => ({
+  chainId: POLYGON_MUMBAI_CHAIN_ID,
+  networkName: 'Polygon Mainnet',
+  networkShortName: 'Polygon',
+  tokenSymbol: 'MATIC',
+  sarcoTokenAddress: '0x80ae3b3847e4e8bd27a389f7686486cac9c3f3e8',
+  diamondDeployAddress: '',
+  etherscanApiUrl: 'https://api.polygonscan.com/api',
+  etherscanApiKey: config?.polygonMainnetApiKey ?? '',
+  explorerUrl: 'https://polygonscan.com/',
+  bundlr: {
+    currencyName: 'ethereum',
+    nodeUrl: 'https://devnet.bundlr.network',
```

Comment:
@DarksightKellar -- can you see what the bundlr URL for polygon is? might just be the same as the eth mainnet one